l10n_it_pec_messages/model/mail_thread.py

```diff
@@ -102,11 +111,72 @@ def get_pec_attachments(self, cr, uid, message, context=None):
                     smime = attachment
         return (postacert, daticert, smime)
 
+    def _message_extract_payload_receipt(self, message,
+                                         save_original=False):
+        """Extract body as HTML and attachments from the mail message"""
```

Comment:
E questa gestione non può essere adottata anche in ```message_parse```, ovvero per tutti i messaggi?